### PR TITLE
RA-63

### PR DESCRIPTION
**Pull request by aoller**

### CODE REVIEW

File: VAS.Services/PlayerController.cs [hunk 1]
```diff
@@ -575,6 +575,30 @@ public virtual void FramerateLower ()
...
+		{
+			Time jumpt = new Time { TotalSeconds = -10 };
+			Seek (CurrentTime + jumpt);
+		}
+
+		public virtual void SeekRightLong ()
+		{
+			Time jumpt = new Time { TotalSeconds = 10 };
+			Seek (CurrentTime + jumpt);
+		}
```
Comment (vguzmanp) [inline VAS.Services/PlayerController.cs]: These methods shouldn't be public, and should be refactored in a more generic method or even a lambda.

Comment (vguzmanp) [inline VAS.Services/PlayerController.cs]: Also, the values should be written in constants / configuration variables

File: VAS.Services/PlayerController.cs [hunk 2]
```diff
@@ -352,7 +352,7 @@ public virtual void TogglePlay ()
 				Play ();
 		}
 
-		public virtual bool Seek (Time time, bool accurate, bool synchronous = false, bool throttled = false)
+		public virtual bool Seek (Time time, bool accurate = false, bool synchronous = false, bool throttled = false)
```
Comment (vguzmanp) [inline VAS.Services/PlayerController.cs]: Why do we need the default here?

Comment (aoller) [inline VAS.Services/PlayerController.cs]: Because in derived class it's overrided in this way.

**Comment by vguzmanp** [comment]
👍 Remember to squash the last two commits

**Pull request merged**